languages/python/polar/polar.py

```diff
@@ -54,16 +58,34 @@ def load_file(self, policy_file):
         until a query is made."""
         policy_file = Path(policy_file)
         extension = policy_file.suffix
-        if extension not in (".pol", ".polar"):
-            raise PolarApiException(f"Polar files must have .pol or .polar extension.")
+        if extension not in (".polar"):
+            raise PolarApiException(f"Polar files must have a .polar extension.")
         if not policy_file.exists():
             raise PolarApiException(f"Could not find file: {policy_file}")
```

Comment:
I think we should make the same changes as in the Ruby library: try reading the file, translating ENOENT into the appropriate PolarError, and also avoid reading it twice